omx/gstomxaudioenc.c

```diff
@@ -536,8 +534,6 @@ gst_omx_audio_enc_loop (GstOMXAudioEnc * self)
     if (flow_ret == GST_FLOW_EOS) {
       GST_DEBUG_OBJECT (self, "EOS");
 
-      gst_pad_push_event (GST_AUDIO_ENCODER_SRC_PAD (self),
```

Comment:
Just wondering, why this was removed.

Reply:
Not sure, I already reversed the change.